asv2/README.md

```diff
@@ -0,0 +1,111 @@
+
+# Federated Attestations ("ASv2") Beta
+
+ASv2 is an open source protocol that maps off-chain personal **identifiers** (such as phone numbers, twitter handles, etc.) to on-chain account **addresses**. This enables a convenient and interoperable user experience for use cases such as:
+
+* payments - send money directly to your friend's phone number!
+* social discovery - find someone's account based on their twitter!
+* any other identity applications!
+
+Here is a short demo of a payment from a user using [Kaala](https://kaala.app/) wallet to a user using [Libera](https://medium.com/impactmarket/ready-to-unlock-your-potential-meet-libera-your-new-crypto-wallet-d1053f917b95) wallet, using only a phone number:
+
+[<img width="800" alt="image" src="https://user-images.githubusercontent.com/46296830/207285114-6ef73be4-10f2-4afc-a066-811e1f3e1042.png">](https://www.loom.com/share/8afddd73ba324ec18aeb63fc96d568f9)
+
+## 🛠 How it Works
+
+ASv2 uses a federated model, meaning that anyone has the power to be an **issuer** of attestation mappings. Issuers have the freedom to decide how to verify that the user actually has ownership of their identifier. After verification, issuers register the mapping as an attestation to the [on-chain smart contract registry](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/identity/FederatedAttestations.sol). Attestations are stored under the issuer that registered them. When looking up attestations, we then have to decide which issuers are trusted.
+
+Here are some active issuers verifying and registering attestations:
+
+| Issuer Name | Address|
+|-----|-----|
+|Kaala| 
+|Libera|
+|Valora
```

Comment:
Could you please:
1. remove Valora, add 
2. add addresses from Kaala and Libera wallet (I asked for them on Discord)